src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
```

Comment:
we should drop this

Reply:
This is to reassure the user that the correct piece of code has been executed. Chrome does it in a similar fashion.